draft-ietf-quic-recovery.md

```diff
@@ -897,9 +897,29 @@ congestion window, which is recommended to be the minimum of
 max_datagram_size is the current maximum size of a datagram for the connection,
 not including UDP or IP overhead.
 
-As an example of a well-known and publicly available implementation of a flow
-pacer, implementers are referred to the Fair Queue packet scheduler (fq qdisc)
-in Linux (3.11 onwards).
+Endpoints can implement pacing as they choose, as long as the rate of bytes
+sent over any interval, `t[n] - t[m]`, does not exceed a small multiple, `N`,
+of maximum value of the congestion window, `congestion_window`, averaged over
+the round trip time estimate, `smoothed_rtt`. Allowing for bursts produces the
+following relation for the total number of bytes sent over any interval:
+
+~~~
+sent@t[n] - sent@t[m]
+    <= N * congestion_window * (t[m] - t[n]) / smoothed_rtt
+       + min(10 * max_datagram_size,
+             max(2 * max_datagram_size, 14720))
+~~~
+
+The inclusion of the factor, `N`, that increases the rate of sending is to
+anticipate increases in sending rate as the congestion window expands. Sending
+at a higher rate also avoids having scheduling delays reduce the send rate
+below that allowed by the congestion controller. A value of 2 ensures that
+pacing is unlikely to limit what is sent any more than the congestion
+controller.
```

Comment:
```suggestion
The inclusion of the factor, `N`, with a value of at least 1 increases the rate
of sending. The value of `N` might be increased to avoid having scheduling
delays reduce the send rate below that allowed by the congestion controller.
```